packages/hooks/src/useEventLogger.ts

```diff
@@ -60,3 +60,47 @@ export function useEventLoggerReducers(
 
   return [eventReducers, setEventReducers]
 }
+
+type UseWrappedEventHandlerProps = {
+  isActive?: boolean
+  name: string
+  handler?: (any) => void
+  addToPayload?: (event) => void
+}
+
+export function useWrappedEventHandler({
+  name,
+  handler,
+  addToPayload,
+  isActive = true,
```

Comment:
`isActive` doesn't seem like the most descriptive prop. Maybe `isLoggingEvents` or something like that?